tutorials/chapter1.md

```diff
@@ -385,22 +422,25 @@ int main() {
 
       eventloop.getSubscriber<float>()->registerListener("mod", [](const float & message) {
             std::cout << "Module received: " << message << std::endl;
+	    std::cout << "Eventloop (subscriber - mod) thread ID: " << std::this_thread::get_id() << std::endl;
          }
       );
       std::thread t([&vectorPublisher]() {
           for (int i = 0; i < 100; i ++) {
             std::vector<float> vector(10);
             for (int j = 0; j < 10; j++) {
-               vector[i] = static_cast<float>(j);
+               vector[j] = static_cast<float>(j);
             }
             vectorPublisher->publish(vector);
             std::this_thread::sleep_for(std::chrono::milliseconds(5));
+	    std::cout << "t (publisher) thread ID: " << std::this_thread::get_id() << std::endl;
```

Comment:
`t` is not a great thread name. I know it might be legacy code, but let's put a better name: `vectorPublisherThread`